rskj-core/src/main/java/co/rsk/core/bc/BlockChainImpl.java

```diff
@@ -540,10 +540,13 @@ private void processBest(final Block block) {
         EventDispatchThread.invokeLater(() -> transactionPool.processBest(block));
     }
 
-    private void onBlock(Block block, BlockResult result) {
+    private void onBlock(Block block, BlockResult result, boolean isBest) {
```

Comment:
Just to not repeat the same signature of the listeners, what if we call this method something like `invokeListeners`?